dockerfiles/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This is intended to be run the plugin's root directory. `dockerfiles/run.sh`
```

Comment:
When is this script used?

Reply:
It is like build from source, here, instead of running in local, it will be executed inside the docker. I am planning to update the Developer guide to include this as well. Also, will use the same script without "docker-compose up" to publish the image to docker hub as well.